components/BackgroundChanger.js

```diff
@@ -0,0 +1,22 @@
+class BackgroundChanger {
+  constructor(app) {
+    this.app = app;
+    this.sectionElement = this.app.sectionElement;
+    this.backgrounds = this.app.backgrounds;
+    this.currentBackground = null;
```

Comment:
Move `backgrounds` to `shared/variables.ts` file and import from there. 
Also `this.app` is redudant.
```suggestion
    this.sectionElement = app.sectionElement;
    this.currentBackground = null;
```